packages/nimble-components/src/table-column/base/tests/table-column.fixtures.ts

```diff
@@ -41,20 +43,58 @@ export class TableColumnEmpty extends TableColumn {
     }
 }
 
-export const tableColumnDelegatesClickAndKeydownTag = 'nimble-test-table-column-delegates';
 /**
- * Simple empty table column with 'click' and 'keydown' event delegation for testing
+ * Column validator used by TableColumnValidationTest
+ */
+export class TestColumnValidator extends ColumnValidator<
+readonly ['invalidFoo', 'invalidBar']
+> {
+    public constructor(columnInternals: ColumnInternals<unknown>) {
+        super(columnInternals, ['invalidFoo', 'invalidBar'] as const);
+    }
+
+    public validateFoo(isValid: boolean): void {
+        this.setConditionValue('invalidFoo', !isValid);
+    }
+
+    public validateBar(isValid: boolean): void {
+        this.setConditionValue('invalidBar', !isValid);
+    }
+}
+
+export const tableColumnValidationTestTag = 'nimble-test-table-column-validation';
```

Comment:
Since this element is globally shared it should be added to the `HTMLElementTagNameMap` adn we can avoid the casts below.

example:
https://github.com/ni/nimble/blob/980eb3cc9d18d2c8c855f784180a6fe1925947a3/packages/nimble-components/src/anchor/index.ts#L12-L16